kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/GcpClientBuilderIT.java

```diff
@@ -42,6 +45,16 @@ public void setup() throws Exception {
             bigQueryInstance.create(TableInfo.of(tableId, StandardTableDefinition.newBuilder().build()));
             logger.info("Created table : "+ tableId.toString());
             // table takes time after creation before being available for operations. You may have to wait a few minutes (~5 minutes)
+            // Try to wait for 5 minutes if table is seen.
+            int attempts = 10;
+            while(bigQueryInstance.getTable(tableId) == null && attempts > 0)  {
+                logger.debug("Busy waiting for table {} to appear! Attempt {}", tableId.getTable(), (10-attempts));
+                Thread.sleep(TimeUnit.SECONDS.toMillis(30));
+                attempts--;
+            }
+            if(attempts == 0) {
+                logger.error("Created table is not available");

```

Comment:
do we need to throw an assertion error here?

Reply:
Should be good to add that